src/information-panel/information-panel.scss

```diff
@@ -74,6 +75,7 @@
         width: $iui-icons-large;
         height: $iui-icons-large;
         margin-right: $iui-s;
+        flex-shrink: 0;
```

Comment:
Line 73 is targeting SVG.  On a local copy I changed it to target `&-icon` and then added the class `.iui-information-header-label-icon` to the Icon in the header to fix `flex-shrink` not being applied correctly.  Screenshot prior to my change when the info panel is very narrow:
![Screen Shot 2021-11-09 at 1 11 39 PM](https://user-images.githubusercontent.com/849817/140980609-03b4f486-3301-4c84-8b80-fb7a30219cc2.png)

Reply:
Adding a new class is not a proper fix, because this icon is passed by the user.

But this is because of webcomponents so we don't have this issue in React.

Reply:
Ah, ok.  Should we add the `flex-shrink` in the HTML's head to fix it for the demo page?  Or leave it as is?

Reply:
I'd say leave it as is, it's not a big deal